multi_import/exporter.py

```diff
@@ -3,10 +3,9 @@
 
 class Exporter(object):
 
-    def __init__(self, mappings, queryset, object_resolver):
-        self.mappings = mappings
+    def __init__(self, queryset, serializer_factory):
```

Comment:
Instead of using a serializer factory, I think we should approach it similar to what DRF viewsets do. The exporters (and other similar classes) should simply define a serializer_class, and then there is a get_serializer method defined on the base class. Checkout GenericAPIView in DRF.